cypress/support/commands.js

```diff
@@ -27,14 +27,12 @@
 // TODO: fix bug in Cypress: always encodeURIComponent in call to visit:
 Cypress.Commands.overwrite('visit', (originalVisit, url, options) => {
   let newURL = url.split('/').map(encodeURIComponent).join('/')
-  if (newURL != url) {
+  if (newURL !== url) {
     Cypress.log({
       name: 'visit',
       message:`‼️  Rewriting ${url} -> ${newURL}`
     })
   }
 
   return originalVisit(newURL, options)
-})
-
-/*global Cypress */
+})
```

Comment:
Did your linter automatically make this change?

Reply:
eslint says "Cypress is already a variable" asks me to delete the line